sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitor.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+/**
+ * A convenient class to instantiate an instance of {@link AzureMonitorInstaller}
+ */
+public final class AzureMonitor {
+
+    /**
+     * Construct an intance of {@link AzureMonitorInstaller}
+     * @return a new {@link AzureMonitorInstaller} object
+     */
+    public static AzureMonitorInstaller installer() {
+        return new AzureMonitorInstaller();
+    }
+
```

Comment:
go ahead and add the convenience `install(AutoConfiguredOpenTelemetrySdkBuilder) method here also so we can evaluate that too

Reply:
that is equivalent to the static install method. either ways works. 

two options:
1. AzureMonitor.installer().installer(AutoConfigurationOpenTelemetrySdkBuilder)
2. AzureMonitorInstaller.install(AutoConfigurationOpenTelemetrySdkBuilder)

I personally prefer option 1.

Reply:
this:

```
AzureMonitor.installer().install(sdkBuilder)
```

doesn't seem much more convenient compared to:

```
new AzureMonitorInstaller().install(sdkBuilder)
```

Reply:
What about `AzureMonitorConfig.new().applyTo(sdkBuilder)`?

Reply:
here are some install(() in upstream https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20install(&type=code

applyToSomething(Object), I haven't seen any applyTo(Object). 

How about initialize(Object)? 
https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20initialize(&type=code

Reply:
> here are some install(() in upstream https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20install(&type=code

If we take the `OpenTelemetryDriver.install(openTelemetry)` and `OpenTelemetryAppender.install(openTelemetrySdk)` cases, it seems the opposit logic: `OpenTelemetry` is installed into  `OpenTelemetryDriver`/`OpenTelemetryAppender`